### ISSUE TITLE
MathJaxのjavascriptを使うプラグインの作成

### ISSUE THREAD

**Issue by Hiroyuki-Nagata** [Hiroyuki-Nagata/fswiki#11]
* [MathJax](https://www.mathjax.org/)

* API
    * [Getting Started](http://docs.mathjax.org/en/latest/start.html)

**Comment by Hiroyuki-Nagata** [comment]
* 疑似コード

```html
<!DOCTYPE html>
<html>
<head>
<title>MathJax TeX Test Page</title>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
</script>
<script type="text/javascript" async
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML">
</script>
</head>
<body>
When $a \ne 0$, there are two solutions to \(ax^2 + bx + c = 0\) and they are
$$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$
</body>
</html>
```

**Comment by Hiroyuki-Nagata** [comment]
Implemented with 742806a0b3eea73bd849e3524ed26e810c8e513d
